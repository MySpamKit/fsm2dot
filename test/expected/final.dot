digraph G {
  node [fontname = "Geneva"]
  edge [fontname = "Geneva"]
  splines=polyline
  stop [shape="doublecircle", style="filled", fillcolor=black, label="",xlabel="stop", fixedsize="true", width=0.25]
}