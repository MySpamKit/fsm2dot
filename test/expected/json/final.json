{"final":["stop"],"choices":[],"states":[],"events":[]}